controllers/users.js

```diff
@@ -130,6 +132,38 @@ const getUsers = async (req, res) => {
       }
     }
 
+    // getting user details by discord id if present.
+    const discordId = req.query.discordId;
+
+    const dev = req.query.dev === "true";
+    if (req.query.discordId) {
```

Comment:
can we merge both condition #139 and #140

Reply:
done